db/migrations/20220921043940_product.sql

```diff
@@ -0,0 +1,20 @@
+-- migrate:up
+CREATE TABLE product (
+    id INT NOT NULL AUTO_INCREMENT,
+    name VARCHAR(50) NOT NULL,
+    price INT NOT NULL,
+    standard_unit VARCHAR(200) NOT NULL,
+    tumbnail_url VARCHAR(2000) NOT NULL,
+    description_url VARCHAR(2000) NOT NULL,
+    anti_bio BOOLEAN NOT NULL,
+    categories_id INT NOT NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP, 
+    PRIMARY KEY(id),
+    CONSTRAINT product_id_fkey FOREIGN KEY (categories_id) REFERENCES categories(id)
```

Comment:
```suggestion
    CONSTRAINT category_id_fkey FOREIGN KEY (categories_id) REFERENCES categories(id)
```
외래키 네이밍이 어색합니다. product_id랑은 전혀 상관이없어 보이네요!